### PR TITLE
Add table widget

### PR DESCRIPTION
**Pull request by pramodya1994**

### CODE REVIEW

File: code-coverage-dashboard/widget/widget-frontend/table-widget/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    plugins: [
+        new CopyWebpackPlugin([
+            {from: path.resolve(__dirname, './src/resources/')}
+        ])
+    ],
+    resolve: {
+        extensions: ['.js', '.json', '.jsx', '.scss']
+    }
+};
```
Comment (isharailanga) [inline code-coverage-dashboard/widget/widget-frontend/table-widget/webpack.config.js]: new line

Comment (isharailanga) [inline code-coverage-dashboard/widget/widget-frontend/table-widget/webpack.config.js]: new line

Comment (pramodya1994) [reply code-coverage-dashboard/widget/widget-frontend/table-widget/webpack.config.js]: done

Comment (pramodya1994) [reply code-coverage-dashboard/widget/widget-frontend/table-widget/webpack.config.js]: done

**Pull request merged**